### PR TITLE
Make RFC3066bis field available in templates via rfc_code property

### PR DESCRIPTION
**Pull request by m038**

### CODE REVIEW

**Comment by ahilles107** [comment]
:+1:

File: newscoop/template_engine/metaclasses/MetaLanguage.php [hunk 1]
```diff
@@ -84,4 +85,4 @@ public static function createFromCode($code)
             $languageId = Language::GetLanguageIdByCode($code);
             return new self($languageId);
         }
-    }
\ No newline at end of file
+    }
```
Comment (takeit) [inline newscoop/template_engine/metaclasses/MetaLanguage.php]: new line at the end of file is missing, apart from that :+1:

Comment (m038) [inline newscoop/template_engine/metaclasses/MetaLanguage.php]: if i'm not mistaken, the diff shows i added a new line at the end of the file :)

Comment (ahilles107) [inline newscoop/template_engine/metaclasses/MetaLanguage.php]: @m038  - you are right. There is new line at end.

Comment (takeit) [inline newscoop/template_engine/metaclasses/MetaLanguage.php]: hmm weird, something wrong is with my editor, it shows me there is no new line, ignore my comment for now

**Comment by takeit** [comment]
:+1:

**Pull request merged**